Nighttime-Operations/Observatory/DIMM/ops.rst

```diff
@@ -0,0 +1,194 @@
+.. |author| replace:: *isotuela*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *none*
+
+.. _DIMM-Ops:
+
+######################################
+DIMM Startup, Shutdown and Monitoring
+######################################
+
+The DIMM automatically performs seeing measurements from dusk until dawn. Limited support from the user is expected. 
```

Comment:
"Limited support from the user is expected." -- belongs on it's own line since it's a separate sentence.

Reply:
Repeated comment. 👍